tests/backup/backup_helper.go

```diff
@@ -3467,7 +3468,7 @@ func GetVolumeMounts(AppContextsMapping *scheduler.Context) ([]string, error) {
 }
 
 // UpdateBackupLocationOwnership Updates the backup location ownership
-func UpdateBackupLocationOwnership(name string, uid string, userNames []string, groups []string, accessType api.Ownership_AccessType, ctx context.Context) error {
+func UpdateBackupLocationOwnership(name string, uid string, userNames []string, groups []string, accessType api.Ownership_AccessType, publicAccess api.Ownership_AccessType, ctx context.Context) error {
```

Comment:
Can we change the parameter to accept readable values instead of  `api. Ownership_AccessType` which is int32?

We can do it like this:

```
type OwnershipAccessType int32


const (
	Read OwnershipAccessType = 1
	Write = 2
	Admin = 3
)
```

Then we can create the function like this

```
func UpdateBackupLocationOwnership(name string, uid string, userNames []string, groups []string, userGroupAccess OwnershipAccessType, publicAccess OwnershipAccessType, ctx context.Context) error {
```

Then the caller will pass it like this

```
err = UpdateBackupLocationOwnership(backupLocationName, backupLocationUID, userNames, nil, Read, 0, ctx)
```

Reply:
Done.

https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/2009/console